python/dlr/counter/model_metric.py

```diff
@@ -64,17 +64,13 @@ def push(self, data):
                 ModelMetric.resp_cnt += 1
 
     def stop(self):
-        if self.executor:
-            with self.condition:
-                self.condition.notify_all()
+        with self.condition:
+            self.condition.notify_all()
             ModelMetric._pub_model_metric = False
             self.executor.shutdown(wait=False)
             mod_dict = ModelExecCounter.get_dict()
```

Comment:
Wouldn't `self.condition.notify_all()` notifies the thread in line 42? Why are we trying to publish dictionary items again?